### DISCUSSION TITLE
English - help / documentation

### DISCUSSION THREAD

**Discussion by sticilface** [shining-man/bsc_fw#204] [General]
I've purchased a BSC board, and I'm looking to set it up however I'm struggling with understanding the nature of all the settings as they are all in German.  I appreciate changing language / adding language is a complex affair... 

Is there a chance of getting some annotations / English documentation to at least compare to the web page that is in German to get started? 

Looks like a very interesting project!  thank you.

**Comment by shining-man** [comment]
I understand the difficulties with the language. Something will come in the future, but I don't want to set a date for it yet, as this involves a lot of work.

**Reply by sticilface** [reply]
I will be patient. And cross my fingers I do not go up in smoke using Google translate! (I also have access to a native German or two) so will also use them!

**Reply by shining-man** [reply]
Can you send me an email? Once it's ready, I can send you a test version. 
My email should be listed in my profile.
